src/api/app/controllers/webui/package_controller.rb

```diff
@@ -380,6 +383,18 @@ def lints_list
     end
   end
 
+  def filter_rpmlint(rpmlint_log_file:, repo:, arch:)
```

Comment:
```suggestion
  def filter_rpmlint(rpmlint_log_file, repo, arch)
```
All of them are required parameters